widgetssdk/src/main/java/com/glia/widgets/call/CallState.java

```diff
@@ -328,16 +331,28 @@ public CallState setTransferring() {
                 .createCallState();
     }
 
-    public boolean isMuteButtonEnabled() {
-        return (isAudioCall() || isVideoCall()) && !showOnHold();
+    public ViewState getGetMuteButtonViewState() {
```

Comment:
Looks like "Get" is unnecessary here, isn't it?
`getGetMuteButtonViewState()` => `getMuteButtonViewState()`

Reply:
Could you also please explain, where is this new function used? Cannot find the usage in the current PR. 🤔

Reply:
It is called from CallView LN:968